frontend/src/metabase/entities/databases/forms.js

```diff
@@ -180,76 +180,87 @@ function getAuthCodeEnableAPILink(engine, details) {
   }
 }
 
-function getFieldsForEngine(engine, details, id) {
-  let info = (MetabaseSettings.get("engines") || {})[engine];
-  if (engine === "bigquery") {
+function getEngineInfo(engine, details, id) {
```

Comment:
Is this refactoring/clean-up related to the whitespace trim? If not, then it's better to keep it as a separate PR to avoid any potential confusion (e.g. from any future `git log` or `git blame`).

Reply:
Moved my refactoring to a new branch/PR -- https://github.com/metabase/metabase/pull/14610 -- and rebased this branch to target `dj-12972-refactor`. Once that PR has merged, I will retarget this PR to master.